.github/workflows/wda-package.yml

```diff
@@ -0,0 +1,48 @@
+name: Building WebDriverAgent
+
+on:
+  workflow_dispatch:
+  pull_request:  # for testing
+
+jobs:
+  build:
+    name: Build and analyse default scheme using xcodebuild command
+    runs-on: macos-latest
```

Comment:
I'm not sure setting macos to latest would work with xcode version provided. This xcode version would probably be dropped from the next macOS, so the job would eventually fail. It makes sense to explicitly provide macOS version for such case and bump them both if/when upgrade is needed

Reply:
I've set as 11 right now